test/integration/apiserver/admissionwebhook/timeout_test.go

```diff
@@ -303,11 +308,19 @@ func testWebhookTimeout(t *testing.T, watchCache bool) {
 				if err == nil {
 					t.Fatalf("expected error but got none")
 				}
-				if tt.errorContains != "" {
-					if !strings.Contains(err.Error(), tt.errorContains) {
-						t.Errorf("expected an error saying %q, but got: %v", tt.errorContains, err)
+
+				expected := false
+				if len(tt.errorContains) != 0 {
+					for _, errStr := range tt.errorContains{
+						if strings.Contains(err.Error(), errStr){
+							expected = true
+							break
+						}
 					}
 				}
+				if !expected{
+					t.Errorf("expected an error saying %q, but got: %v", tt.errorContains, err)
```

Comment:
nit: `expected the error to be any of %q, but got: %v`

Reply:
done~ PTAL